tests/v2_validation/cattlevalidationtest/core/test_webhook.py

```diff
@@ -0,0 +1,116 @@
+from common_fixtures import *  # NOQA
+import requests
+from requests.auth import AuthBase
+import json
+
+access_key = os.environ.get("CATTLE_ACCESS_KEY", 'admin')
+secret_key = os.environ.get("CATTLE_SECRET_KEY", 'admin')
+
+def test_construct_webhook_test_max(client):
+    # Create Environment
+    env = client.create_stack(name=random_str())
+    env = client.wait_success(env)
+    assert env.state == "active"
+
+    # Create Service of scale 1
+    service_name = random_str()
+
+    image_uuid = TEST_IMAGE_UUID
+    launch_config = {"imageUuid": image_uuid}
+    service = client.create_service(name=service_name,
+    	                            stackId=env.id,
+                                    launchConfig=launch_config)
+    service = client.wait_success(service)
+    assert service.state == "inactive"
+    service.activate()
+    service = client.wait_success(service)
+    assert service.state == "active"
+
+    # Create webhook to scale up service by 2
+    url = "http://localhost:8080/v1-webhooks/receivers?projectId="+env.accountId
```

Comment:
you cant hard code the URL to localhost. you should be able to find it as a property on the client